dashboard/src/containers/App.js

```diff
@@ -1,18 +1,24 @@
 import React from "react";
 import { connect } from "react-redux";
 import screenfull from "screenfull";
+import { Alert } from "react-bootstrap";
 
-import ScratchpadContainer from "../containers/ScratchpadContainer";
-import NavBar from "../components/Navbar";
+import NavbarContainer from "../containers/NavbarContainer";
 import PreviousQueryListContainer from "./PreviousQueryListContainer";
 import Editor from "./Editor";
 import Response from "./Response";
-import { updateFullscreen } from "../actions";
+import {
+  updateFullscreen,
+  getQuery,
+  updateInitialQuery,
+  queryFound,
+  initialServerState
+} from "../actions";
 
 import "../assets/css/App.css";
 
-class App extends React.Component {
-  enterFullScreen = updateFullscreen => {
+const App = React.createClass({
+  enterFullScreen(updateFullscreen) {
```

Comment:
Any reasons for no going back to `createClass`? I feel that the previous implementation does the same thing with ES6 class and autobinding with arrow function.